### DISCUSSION TITLE
No interface for callback function?

### DISCUSSION THREAD

**Discussion by xpt1x** [srivastavaanurag79/react-native-paper-select#22] [General]
Is there any special reason for leaving callback fn untyped ?
Because it can simply be
```ts
export interface SelectedItem {
  text: string;
  selectedList: list;
}
// and
onSelection: (item: SelectedItem) => void;
```

**Comment by srivastavaanurag79** [comment]
I guess forgot about it as there is only one callback function, will add it for sure.

Thanks

**Comment by srivastavaanurag79** [comment]
> Is there any special reason for leaving callback fn untyped ? Because it can simply be
> 
> ```ts
> export interface SelectedItem {
>   text: string;
>   selectedList: list;
> }
> // and
> onSelection: (item: SelectedItem) => void;
> ```

```
export interface SelectedItem {
  text: string;
  selectedList: Array<list>;
}
```
